### PR TITLE
vtc compatible high-volubility sampler

### PR DESCRIPTION
**Pull request by lucasgautheron**
- [ ] tests
- [ ] documentation
- [ ] implementation
    - [x] VTC compatible cvc
    - [x] VTC compatible ctc

### CODE REVIEW

**Comment by lucasgautheron** [comment]
VTC/LENA comparison using tsimane2017

AWC

Screenshot 2021-03-16 at 23 02 15

CVC
Screenshot 2021-03-16 at 22 53 54

CTC

Screenshot 2021-03-16 at 22 47 22

File: ChildProject/pipelines/samplers.py [hunk 1]
```diff
@@ -372,16 +372,32 @@ def _segment_scores(self, recording):
         windows['recording_filename'] = recording['recording_filename']
 
         if self.metric == 'ctc':
-            # NOTE: The original code explicitly chooses for these conversational turn 
-            # types, but we might wish to verify what they are. 
-            segments['is_CT'] = segments['lena_conv_turn_type'].isin(['TIFR', 'TIMR'])
+            if 'lena_conv_turn_type' in segments.columns:
+                # NOTE: This is the equivalent of CTC (tab1) in rlena_extract.R
+                segments['is_CT'] = segments['lena_conv_turn_type'].isin(['TIFR', 'TIMR'])
+            else:
```
Comment (lucasgautheron) [inline ChildProject/pipelines/samplers.py:378]: do the following lines really mimic LENA's cvc ?

File: ChildProject/pipelines/samplers.py [hunk 2]
```diff
@@ -372,16 +372,32 @@ def _segment_scores(self, recording):
...
+                segments.sort_values(['segment_onset', 'segment_offset'])
+                segments = segments[segments['speaker_type'].isin(['FEM', 'MAL', 'CHI'])]
+                segments['duration'] = segments['segment_offset']-segments['segment_onset']
+                segments['iti'] = segments['segment_onset'] - segments['segment_offset'].shift(1)
+                segments['prev_speaker_type'] = segments['speaker_type'].shift(1)
+
+                key_child_env = ['FEM', 'MAL', 'CHI']
+                segments['is_CT'] = segments.apply(
+                    lambda row: (row['iti'] < 1000) and (
```
Comment (lucasgautheron) [inline ChildProject/pipelines/samplers.py:387]: I set this threshold arbitrarily, but what should we do ?

Comment (lucasgautheron) [reply ChildProject/pipelines/samplers.py:387]: See #174

File: ChildProject/pipelines/samplers.py (hunk 1 above)
Comment (alecristia) [reply ChildProject/pipelines/samplers.py:382]: do you mean cvc or ctc? the code displayed above lines 375-8 is for CTC, right?

**Comment by lucasgautheron** [comment]
Here they claim that the LENA overestimates CTC, so the VTC seems even worse in that respect.. https://srcd.onlinelibrary.wiley.com/doi/10.1111/cdev.13495

**Pull request merged**